### PR TITLE
disabled non-supported template testing

### PR DESCRIPTION
**Pull request by jim-bo**

### CODE REVIEW

**Comment by jim-bo** [comment]
@curlup minor commit to disable testing of non-supported templates...

File: cidc_schemas/prism.py [hunk 1]
```diff
@@ -352,7 +352,7 @@ def _process_property(
         return files
 
 
-SUPPORTED_ASSAYS = ["wes", "olink", "cytof"]
```
Comment (curlup) [inline cidc_schemas/prism.py]: Are we sure we want to regress on that?

File: cidc_schemas/template.py [hunk 2]
```diff
@@ -42,30 +42,6 @@ def generate_empty_template(schema_path: str, target_path: str):
     template.to_excel(target_path)
 
 
-def generate_all_templates(target_dir: str):
```
Comment (jacoblurye) [inline cidc_schemas/template.py]: This function is used in the cidc schemas CLI (and in turn by the CIDC Portal for generating downloadable templates). Why is it deleted?

https://github.com/CIMAC-CIDC/cidc-schemas/blob/9d4d6b6cb86d4d96c6070cce232c012c3996b9c0/cidc_schemas/cli.py#L97

File: tests/test_template.py [hunk 3]
```diff
@@ -80,8 +81,42 @@ def test_worksheet_processing():
     assert Template._process_worksheet(worksheet) == target
 
 
+def generate_all_templates(target_dir: str):
```
Comment (jacoblurye) [inline tests/test_template.py]: This should be imported from `cidc_schemas.template`, not copied here.

File: tests/test_templates.py [hunk 4]
```diff
@@ -10,6 +10,7 @@
 from cidc_schemas.template import Template
 from cidc_schemas.template_writer import RowType
 from cidc_schemas.template_reader import XlTemplateReader, ValidationError
+from cidc_schemas.prism import SUPPORTED_ASSAYS, SUPPORTED_MANIFESTS, SUPPORTED_TEMPLATES
```
Comment (jacoblurye) [inline tests/test_templates.py]: Only `SUPPORTED_TEMPLATES` appears to be used.

**Pull request closed without merging**